app/data_providers/dispatch.py

```diff
@@ -25,6 +27,27 @@
 from app.models.request_user import RequestUser
 from app.rankers.algorithms import rank_by_preferred_topics, spread_topics
 
+# todo: add thompson sampling
+class Item2ItemDispatch:
+
+    def __init__(self,
+                 # metrics_client: MetricsFetchable,
+                 item_recommender: Item2ItemRecommender):
+        # self.metrics_client = metrics_client
+        self.item_recommender = item_recommender
+
+    async def syndicated(self, resolved_id: int, count: int) -> List[CorpusRecommendationModel]:
```

Comment:
Nit: I always use verbs for functions, e.g. `recommend_syndicated`, but the code is readable as-is, so I don't feel strongly here.

Reply:
long names are not pythonic, I think it's fine to have non-verbs if it's readable